src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -763,14 +755,59 @@ int wm_vuldet_send_agent_report(sqlite3 *db, OSHash *cve_table, agent_software *
             int bugref_count = 0;
 
             if (pkg->feed & VU_SRC_NVD) {
+
+                os_strdup("NVD feed", report->alert_origin);
+
+                if (pkg->nvd_cond) {
+                    if (pkg->nvd_cond->fix_version) w_strdup("Fixed", report->state);
+                    report->pending = pkg->nvd_cond->fix_version ? 0 : 1;
+
+                    // building condition
```

Comment:
The condition cover four cases:

1. Package between two versions
2. Package greater/equal than
3. Package less/equal than
4. A vulnerability that matches all the packages versions.